lib/LedgerNodeWorkSession.js

```diff
@@ -31,46 +31,40 @@ module.exports = class LedgerNodeWorkSession {
   }
 
   /**
-   * Reserves the given ledgerNode for a work session, executes a function
-   * to perform work on it, and then releases the ledgerNode from reservation
-   * so that other work sessions may be used to reserve it.
+   * Reserves the given ledgerNode for a work session, schedules work to
+   * perform on the next tick, and returns a promise that resolves to true
+   * if the reservation was successful and false if the reservation fails
+   * because the ledgerNode was reserved by another work session.
    *
-   * The ledgerNode will be reserved if has not been marked as deleted, and it
-   * has no existing work session.
+   * On the next tick, the given function will be called to perform work on the
+   * ledger node, and once it completes, the ledgerNode will be released from
+   * this work session so that other work sessions may be used to reserve it.
    *
-   * @param maxAge the maximum time to reserve the ledgerNode for.
-   * @param fn(session, callback) the work function to execute.
+   * The ledgerNode will be successfully reserved if has not been marked as
+   * deleted, and it has no existing work session.
+   *
+   * @param options The options to use.
```

Comment:
this is not a major issue, but it should be something like:

```js
@param {object} options - The options to use.
```

If time is important, this can be cleaned up on release.

Reply:
The whole module's docs are out of date, skipping this for now. Should be fixed up all at once.

Reply:
@dlongley issue added: https://github.com/digitalbazaar/bedrock-ledger-node/issues/80